lib/internal/cluster/primary.js

```diff
@@ -221,12 +221,12 @@ function emitForkNT(worker) {
 }
 
 cluster.disconnect = function(cb) {
-  const workers = ObjectKeys(cluster.workers);
+  const workers = ObjectValues(cluster.workers);
 
   if (workers.length === 0) {
     process.nextTick(() => intercom.emit('disconnect'));
   } else {
-    for (const worker of ObjectValues(cluster.workers)) {
```

Comment:
Is https://github.com/nodejs/node/blob/main/doc/contributing/primordials.md#unsafe-array-iteration applicable here?